client/verta/verta/monitoring/summaries/queries.py

```diff
@@ -215,7 +215,7 @@ def __init__(
         self._labels = maybe(_labels_proto, labels)
         self._time_window_start = time_window_start
         self._time_window_end = time_window_end
-        self._aggregation = aggregation
+        self.aggregation = aggregation
```

Comment:
Is this overriding the `@property` `aggregation` on line 247?

Reply:
No, and in fact it should be causing the initializer to defer to the property setter. This way `self.aggregation = aggregation` invokes the validation logic baked into that setter.

Reply:
```
In [1]: from verta.monitoring.summaries.queries import SummaryQuery, SummarySampleQuery
   ...:

In [2]: from verta._protos.public.monitoring.Summary_pb2 import (
   ...:     AggregationQuerySummary,
   ...:     FilterQuerySummarySample,
   ...:     FindSummaryRequest,
   ...:     FindSummarySampleRequest,
   ...:     LabelFilterQuerySummarySample,
   ...: )
   ...:

In [3]: empty_agg_proto = AggregationQuerySummary()

In [4]: q = SummarySampleQuery(aggregation=empty_agg_proto)

In [5]: q
Out[5]:
SummarySampleQuery(filter {
  find_summaries {
    page_number: 1
    page_limit: -1
  }
}
page_number: 1
page_limit: -1
)
```

Reply:
Ah yes I see it now!